ydb/core/protos/whiteboard_disk_states.proto

```diff
@@ -0,0 +1,18 @@
+package NKikimrWhiteboard;
+
+enum EFlag {
```

Comment:
this is not a "disk status". this is a common status, shared by many parts of our system. it shouldn't be in "disk_states.proto"